frontend/components/site/CommitSummary.jsx

```diff
@@ -66,7 +66,7 @@ CommitSummary.propTypes = {
     branch: PropTypes.string.isRequired,
     username: PropTypes.string.isRequired,
     clonedCommitSha: PropTypes.string.isRequired,
-    createdAt: PropTypes.instanceOf(Date).isRequired,
+    createdAt: PropTypes.string.isRequired,
```

Comment:
This was creating an error in the console. I know it "should" be a date but it's not by the time it gets here.